tests/validation/tests/v3_api/test_airgap.py

```diff
@@ -299,11 +390,89 @@ def prepare_airgap_node(bastion_node, number_of_nodes):
 
 
 def deploy_airgap_nodes(bastion_node, ag_nodes):
```

Comment:
What is the purpose of this method? Including comments above each method to describe the purpose would be helpful

Reply:
Oops, I changed some things around with that function so that was supposed to get deleted